Makefile

```diff
@@ -60,4 +53,3 @@ endif
 
 clean:
 	rm -f nginx-ingress
-	rm -f Dockerfile
```

Comment:
why was it removed?

Reply:
Instead of copying Dockerfiles to the root of the project I point `docker build` directly to the Dockerfile required:
https://github.com/nginxinc/kubernetes-ingress/pull/825/commits/baea83e6e0e100e9c18870eea173dd76ef4a8c49#diff-b67911656ef5d18c4ae36cb6741b7965R43

So there is no trailing Dockerfile to remove/cleanup